docs/code-deployment-flow.md

```diff
@@ -31,6 +31,25 @@ PNPM Monorepos use symlinks between workspace node_modules and the top level nod
 *bundlingRequired* = `Stack.of(this).bundlingRequired` [see](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.Stack.html#bundlingrequired)
 *skipBuild* = `NextjsProps.skipBuild`
 
+## CloudFront Distribution Policies Quotas
+
+There is a hard limit of 20 Response headers policies per AWS account and Cache policies per AWS account  and when deploying multiple sites this limit is easily reached.
+
+For this reason you can optionally pass in your own cache policies and response headers policies to the `Nextjs` construct.
+
+```ts
+new Nextjs(this, 'Nextjs', {
+  nextjs: {
```

Comment:
Thank you for documenting! Could you update this to use `NextjsProps` instead of `NextjsOverrides`?

Reply:
I may not be following what you mean, could you show me?

Reply:
well... I'm not sure now, too many options for passing props, let me try to move it up and see how it goes code-wise.